templated_email/backends/vanilla_django.py

```diff
@@ -113,12 +113,16 @@ def _render_email(self, template_name, context,
 
         return response
 
-    def get_email_message(self, template_name, context, from_email=None, to=None,
+    def get_email_message(self, template_name, context=None, from_email=None, to=None,
                           cc=None, bcc=None, headers=None,
                           template_prefix=None, template_suffix=None,
                           template_dir=None, file_extension=None,
                           attachments=None, create_link=False):
 
+        from_email = from_email or settings.DEFAULT_FROM_EMAIL
```

Comment:
We have also a `TEMPLATED_EMAIL_FROM_EMAIL` setting.
A user thought this was a default. See: https://github.com/vintasoftware/django-templated-email/issues/97#issuecomment-376885819

So I think this should read: `from_email = from_email or settings.TEMPLATED_EMAIL_FROM_EMAIL or settings.DEFAULT_FROM_EMAIL`

Also, check again the README, see if this is well explained. And check where `TEMPLATED_EMAIL_FROM_EMAIL` is being used.

Reply:
From what I've seen, this `TEMPLATED_EMAIL_FROM_EMAIL` is used only by the Class-based mixin. It's documentation is written [here](https://github.com/vintasoftware/django-templated-email#class-based-views). While it's not saying explicitly that the setting is not required, it does not specify that it'll fall back to `DEFAULT_FROM_MAIL`. I think this could be updated.

I'm not sure, however, if we should put it as a default in the vanilla-django file, since this class-based mixin is like a wrapper over the vanilla.

Reply:
I think we should make the backend use this setting `TEMPLATED_EMAIL_FROM_EMAIL`. So `from_email = from_email or settings.TEMPLATED_EMAIL_FROM_EMAIL or settings.DEFAULT_FROM_EMAIL` seems fine for me.

Reply:
@fjsj I've updated the vanilla backend and added a note about this behavior in the readme.